### ISSUE TITLE
HunFlair: possibility of training to recognise additional entity types?

### ISSUE THREAD

**Issue by MagdalenaMladenova** [flairNLP/flair#2673]
Hi, 

If I understand correctly, HunFlair includes 5 different models wrapped together, each recognising a single biomedical entity type. I saw your useful tutorial on training/fine-tuning to improve performance on these specific entities. 

However, I was wondering - is there an established pipeline for training a model (with an identical architecture to the ones described in your paper) to recognise a new entity type? I can prepare my own dataset with labels for additional entity types (e.g. cell type /_neuron_/, cell process /_apoptosis_/), but I'm not sure what's the best way to use your platform for this.

Thank you in advance for any help!

**Comment by leonweber** [comment]
Thanks for your question! As you pointed out, HunFlair runs the models for the five different entity types independently from each other. So, adding another model boils down to run another model next to the five HunFlair models.

To train HunFlair for a new entity type, please follow [this tutorial](https://github.com/flairNLP/flair/blob/master/resources/docs/HUNFLAIR_TUTORIAL_2_TRAINING.md#train-a-biomedical-ner-model-from-scratch). The architecture and hyperparameters described in this section are the same that we used for preparing the HunFlair models.

Loading your dataset for training is described [here](https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_6_CORPUS.md#reading-your-own-sequence-labeling-dataset).
